.github/workflows/ci.yml

```
@@ -13,9 +13,15 @@ jobs:
           - runner: ubuntu-latest
             os: Linux
             arch: x86_64
+            clang-version: 18
```

It's a bit more convenient if you do it as


```suggestion
            clang-version: clang-18
```

so the default name says `clang-18` instead of just `18`.

---

Oh, I didn't realize. Let's just do that then.